### DISCUSSION TITLE
ProgressBar with a Spinner

### DISCUSSION THREAD

**Discussion by samsja** [Textualize/rich#2014] [Q&A] [answered]
Hey all,

I was wondering if it was possible to add a Spinner in a ProgressBar.

The final objective is to have something like this:
![Peek 2022-03-01 14-53](https://user-images.githubusercontent.com/55492238/156181706-49ef55da-aabe-4a9c-be4a-82bb828139f7.gif)

Thanks in advance !

**Comment by willmcgugan** [comment, marked as answer]
See `SpinnerColumn` https://rich.readthedocs.io/en/latest/progress.html#columns

**Reply by samsja** [reply]
Thank you that is what I was looking for
